deploy/example.yaml

```diff
@@ -0,0 +1,207 @@
+
+# This configuration is for OpenEBS Installation, ControlPlaneUpgrade
+# and unInstallation.
+apiVersion: dao.mayadata.io/v1alpha1
+kind: OpenEBS
+metadata:
+  name: install-openebs-1.5
+  # Namespace i.e. the namespace where the OpenEBS operator and the
+  # other openebs components will be installed/needs-to-be-installed.
+  #
+  # Defaults to “openebs” namespace.
+  namespace: openebs
+  labels:
+    app.kubernetes.io/name: openebs-operator
+  annotations:
+spec:
+  # OpenEBS Version.
+  #
+  # Defaults to the latest(for instance 1.5) OpenEBS version
+  version: "1.5"
+  # defaultStoragePath is the directory which will be used by
```

Comment:
Please leave a line space to make this YAML readable.
Please do this across all the yamls